src/app/components/Unlock/index.jsx

```diff
@@ -33,20 +35,26 @@ class Unlock extends React.Component {
 
   render() {
     return (
-      <div>
+      <div class="unlock--container">
```

Comment:
do we follow some kind of naming convention here? 
what are those?

Reply:
It's a scheme/ convention called BEM (Block, Element, Modifier)
http://getbem.com/introduction/